.github/workflows/tests.yml

```diff
@@ -8,4 +8,4 @@ jobs:
     - uses: actions/setup-ruby@v1
       with:
         ruby-version: '2.6'
-    - run: ruby tests/well-formed-yaml.rb
+    - run: 'ruby tests/well-formed-yaml.rb && ruby tests/all-links-are-ok.rb'
```

Comment:
I'd prefer each of these scripts to be separate `run` steps, please. Then we get individual feedback on each of the separate tasks it's doing.